File: java/AddTwoIntegers.java
public class AddTwoIntegers {
    public static int sum(int num1, int num2) {
        return num1 + num2;
    }

    public static String stringSum(String num1, String num2) {
        return String.valueOf(Integer.parseInt(num1) + Integer.parseInt(num2));
    }

    public static void main(String[] args) {
        System.out.println(sum(12, 5));     // 17
        System.out.println(sum(-10, 4));    // -6
        System.out.println(sum(-456, 77));  // -533
        System.out.println(sum(0, 0));      // 0

        System.out.println();

        System.out.println(stringSum("12", "5"));      // 17
        System.out.println(stringSum("-10", "4"));     // -6
        System.out.println(stringSum("-456", "77"));   // 533
        System.out.println(stringSum("0", "0"));       // 0
    }
}

File: go/add_two_integers.go
package main

import "fmt"

// Given two integers num1 and num2, return the sum of the two integers.
func sum(num1 int, num2 int) int {
    return num1 + num2
}

func main() {
    fmt.Println(sum(12, 5))     // 17
    fmt.Println(sum(-10, 4))    // -6
    fmt.Println(sum(-456, 77))  // -379
    fmt.Println(sum(0, 0))      // 0
}
